schedule/functional/autoyast_create_hdd/autoyast_create_hdd_gnome_uefi.yaml

```diff
@@ -0,0 +1,16 @@
+name: autoyast_create_hdd_gnome_uefi
+description:    >
+  Test performs autoyast installation to generate qcow images used by dependent functional tests.
+vars:
+  AUTOYAST: create_hdd/create_hdd_%DESKTOP%_uefi_%ARCH%.xml
+schedule:
```

Comment:
I guess the `uefi` part should be after x86_64, or the architecture: `create_hdd/create_hdd_%DESKTOP%_%ARCH%_uefi.xml`